tests/numpy/test_arithmetic.py

```diff
@@ -126,9 +144,48 @@ def test_inner_outer(nps_app_inst):
     assert np.allclose(np.inner(A, B), nps.inner(nps_A, nps_B).get())
     assert np.allclose(np.outer(A, B), nps.outer(nps_A, nps_B).get())
 
+def test_broadcast(nps_app_inst):
+    import numpy as np
+    from nums import numpy as nps
+    import pytest
+    assert nps_app_inst is not None
+
+    def check_matrix_broadcast_mismatch_simple(_np_a, _np_b):
```

Comment:
Can we rename this to `check_broadcast_mismatch_simple`? The `matrix` in the function name suggests there's a matrix multiply being tested, but it's element-wise ops on 2d arrays.